docs/user/openstack/provider_networks.md

```diff
@@ -0,0 +1,123 @@
+# Provider Networks
+
+
+## Table of Contents
+- [Provider Networks](#provider-networks)
+  - [Table of Contents](#table-of-contents)
+  - [Introduction](#introduction)
+  - [Common prerequisites](#common-prerequisites)
+  - [Considerations when creating provider networks](#considerations-when-creating-provider-networks)
+  - [Deploying cluster with primary interface on a provider network with IPI](#deploying-cluster-with-primary-interface-on-a-provider-network-with-ipi)
+  - [Known issues](#known-issues)
+
+
+## Introduction
+
+Provider networks map directly to an existing physical network in the data center.
+Useful network types in this category include flat (untagged), VLAN (802.1Q tagged) and VXLAN. 
+OpenShift clusters can be deployed on provider networks, so the installer won't need to create
+tenant networks nor floating IPs.
+More information can be found about provider networks terminology [here][1].
+
+Here is a basic architecture of one OCP cluster running on a provider network and another one
+on a tenant network:
+
+![OCP on a provider network](provider-network.png)
+
+
+## Common prerequisites
+
+* [Neutron service][2] is enabled and accessible through the [OpenStack Networking API][3].
+* Neutron should be configured with the [port-security and allowed-address-pairs][4] extensions so the installer can
+  add the `allowed_address_pairs` attribute to ports.
+* The Provider Network has to be created by the administrator with specific configuration (see the next section).
+* The Metadata IP (169.254.269.254) has to be routed for the provider network and we need to enable the DHCP server
+  for the subnet (see the next section).
+
+
+## Considerations when creating provider networks
+
+* The provider network has to be owned by the project ID used to bootstrap the OCP cluster.
+
+    It is important that the provider network and subnet are owned by the same project that will be used
```

Comment:
This is a good opportunity to use "must," I think. 

"The same project that you use to install OpenShift must own the provider network and subnet."

Or something like that.